openfisca_tools/model_api.py

```diff
@@ -136,8 +226,38 @@ def random(entity, reset=True):
     return x
 
 
-def is_in(values, *targets):
-    return sum(map(lambda target: values == target, targets))
+def is_in(values: ArrayLike, *targets: list) -> ArrayLike:
+    """Returns true if the value is in the list of targets.
+
+    Args:
+        values (ArrayLike): The values to test.
+
+    Returns:
+        ArrayLike: True if the value is in the list of targets.
+    """
+    if (len(targets) == 1) and isinstance(targets[0], list):
+        targets = targets[0]
+    return np.any([values == target for target in targets], axis=0)
+
+
+def between(
+    values: ArrayLike, lower: float, upper: float, inclusive: bool = True
+) -> ArrayLike:
+    """Returns true if values are between lower and upper.
+
+    Args:
+        values (ArrayLike): The input array.
+        lower (float): The lower bound.
+        upper (float): The upper bound.
+        inclusive (bool, optional): Whether to include or exclude the bounds. Defaults to True.
```

Comment:
Could you make this a string like in [`pd.between`](https://pandas.pydata.org/docs/reference/api/pandas.Series.between.html)?

For example, in WIC we'll want to classify children as `between(person("age", period), 1, 5, inclusive="left")`.

Reply:
OK yeah, just used Pandas functionality here